scripts/RNA_analysis/make_complexity_plots.py

```diff
@@ -27,12 +28,13 @@
     extra_arg = config['sbatch']['extra_arg']
     tools   = config['custom_algorithms']['RNA-seq analysis']
     bam     = tools['bamtools']+'/'+tools['bamtools_version']
+    bed     = tools['BEDTools']+'/'+tools['BEDTools_version']
     preseq  = tools['Preseq']+'/'+tools['Preseq_version']
 except:
-    print 'ERROR: problem loading samtools version from config file'
+    print 'ERROR: problem loading tools version from config file'
 
 
-f=open("complexity_"+name+".sh",'w')
+f=open("complexity_"+name+".sh",'wa')
 
 print >>f, """#!/bin/bash -l
```

Comment:
personal opinion: in general I like more to see a print statement for each line rather that a single print for many lines. It is easier to change in the future and easier to debug.